README.md

```diff
@@ -810,6 +810,7 @@ Setup a Terra workspace.
 Commands:
   add-user         Add a user or group to the workspace.
   break-glass      Grant break-glass access to a workspace user.
+  configure-aws    Generate an AWS configuration file for a workspace.
```

Comment:
How about extending the `terra config` or `terra config set` commands?

pro: avoid confusion between `terra config` and `terra configure-aws`
con: `terra config` and `terra configure-aws` address slightly different functionalities. 

alternately 

`terra aws-configure` or
 `terra aws configure`?  // this makes it seem like aws is a passthrough app command

Reply:
No strong opinions on any of the options. Open to changes later as well if UX prefers a different approach